plugins/public/cinq-collector-aws/cinq_collector_aws/account.py

```diff
@@ -105,31 +105,36 @@ def update_rds_databases(self):
             response = lambda_client.invoke(FunctionName=self.rds_function_name, InvocationType='RequestResponse',
                                             Payload=input_payload
                                             )
+            print(response)
```

Comment:
I suggest we log it if we need to know what's in it

Reply:
yeah left this in, tbh, we don't really need the top level response since we'll always want the detail in the body.